_posts/blog/2019-08-19-Ruslan's-Final-Report.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: GSoC Final Report by Ruslan 
+categories: blog
+excerpt: work product submission on the scona project during GSoC program
+image:
+  feature:
+link:
+date: 2019-08-19
+modified:
+share: true
+author: ruslan_yermakov
+---
+
+## TL;DR - GSoC Summary
+
+> **Student**: Ruslan Yermakov  
+**Mentor**: Dr Kirstie Whitaker    
+**Organization**: [INCF](https://www.incf.org/gsoc2019/projectlist)    
+**Project**: In the course of the program I was working on the open-source project [**scona**](https://github.com/WhitakerLab/scona) - a toolkit to perform brain network analyses. 
+The goal of my project was to produce publication-ready brain network analysis results and visualizations from the command line. 
+
+In this blog I am sharing the achieved deliverables and results with the community.
+
+## The scope of the project
+
+Neuroscientists and neuroimaging researches perform structural network analyses on the MRI data.
+Scona is designed to imrove this experience and to help researchers run brain analyses quickly and reliably in order to capture many aspects of brain structure and function.
+
+My project is mainly focused on the visualisation part that is now included in the scona package.
+  
+Achieved deliverables:
+- implementation of visualization module to plot MRI data and the results of structural covariance brain network analyses;  
+- development of visualisation functions which allows users to produce publication-ready plots based on the results of the performed analysis for publication in a scientific journal;  
+- writing documentation and tutorials to help users better understand how to use the package;
+- fixing bugs and issues in the project; 
+
+
+#### Links to my contributions
```

Comment:
Is this the right level of heading? Maybe 3 ### ? Previous was 2.

Same for headings further down (I'm not going to comment on all of them).

Give the post a render and see what looks best.

Reply:
Yeah, levels of heading:
# H1
## H2
### H3
#### H4
##### H5
###### H6

I do not see any big difference, so I will probably change to ### (3) because the 1st heading was ##(2) - just have 1-level decrease